pkg/providers/tinkerbell/hardware/catalogue_bmc.go

```diff
@@ -1,7 +1,7 @@
 package hardware
 
 import (
-	"github.com/tinkerbell/rufio/api/v1alpha1"
+	rufiov1alpha1 "github.com/tinkerbell/rufio/api/v1alpha1"
```

Comment:
Is this needed if we don't reference our own spec in the file?

Reply:
It's not needed, but I think this is what we should do since we have multiple `v1alpha1` references in our codebase. It makes it easier to tell which one is being used at a glance

Reply:
pls use `rufiov1` :)